docs/reference/esql/functions/types/coalesce.asciidoc

```diff
@@ -1,49 +1,9 @@
 [%header.monospaced.styled,format=dsv,separator=|]
 |===
 arg1 | arg2... | result
-boolean | boolean
 boolean | boolean | boolean
```

Comment:
praise: that is much cleaner and clearer IMO.

Reply:
At some point we'll want this to be able to promote as part of these - like `COALESCE(12, 123432153)` becoming a `LONG` instead of an error. We'll want some way to say that with these examples - and I think something like this is right for it.